packages/server/graphql/public/types/NotifyResponseReplied.ts

```diff
@@ -0,0 +1,24 @@
+import MeetingTeamPrompt from '../../../database/types/MeetingTeamPrompt'
+import {getTeamPromptResponsesByMeetingId} from '../../../postgres/queries/getTeamPromptResponsesByMeetingIds'
+import {NotifyResponseRepliedResolvers} from '../resolverTypes'
+
+const NotifyResponseReplied: NotifyResponseRepliedResolvers = {
+  __isTypeOf: ({type}) => type === 'RESPONSE_REPLIED',
+  meeting: async ({meetingId}, _args, {dataLoader}) => {
+    const meeting = await dataLoader.get('newMeetings').load(meetingId)
+    return meeting as MeetingTeamPrompt
+  },
+  response: async ({userId, meetingId}, _args: unknown, {dataLoader}) => {
+    // TODO: implement getTeamPromptResponsesByMeetingIdAndUserId
+    const responses = await getTeamPromptResponsesByMeetingId(meetingId)
```

Comment:
+1 do you want to create a follow-up task, so this comment won't live forever? 😅

Reply:
😅 created https://github.com/ParabolInc/parabol/issues/7573